### PR TITLE
Create .travis.yml

### PR DESCRIPTION
**Pull request by techtonik**
Check to see if branch local .travis.yml works.
See https://github.com/mingwpy/mingwpy.github.io/issues/33#issuecomment-182778987

### CODE REVIEW

**Comment by matthew-brett** [comment]
I enabled it on travis, I think.

**Comment by techtonik** [comment]
@matthew-brett I believe it needs a commit to bootstrap - https://travis-ci.org/mingwpy/ - merging this one should be fine for a start.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# build documentation in `source` branch and commit result to `master`, from
+# https://github.com/matplotlib/matplotlib/blob/c259a556/.travis.yml#L147-L169
+
+sudo: false
+
+language: python
+
+script:
+  - make.bat html
```
Comment (matthew-brett) [inline .travis.yml]: Isn't this windows specific?

**Comment by matthew-brett** [comment]
Suggest adding

```
python:
    - 2.7

before_install:
    - pip install sphinx
```

**Comment by matthew-brett** [comment]
Oops, then:

```
    - pip install linkchecker
script:
    - make html
    - linkchecker _build/html/index.html
```

File: .travis.yml (hunk 1 above)
Comment (techtonik) [inline .travis.yml]: Yes. Sorry, I am on Windows so I forgot that it is for Travis and not for AppVeyor.

**Comment by techtonik** [comment]
Squashed some commits for clarity.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# build documentation in `source` branch and commit result to `master`, from
+# https://github.com/matplotlib/matplotlib/blob/c259a556/.travis.yml#L147-L169
+
+sudo: false
+
+language: python
+
```
Comment (matthew-brett) [inline .travis.yml:7]: I think you still need:

```
install:
    - pip install -r requirements.txt
```

Comment (techtonik) [inline .travis.yml:7]: No. Click the green commit checkmark or expand "All checks passed" box to see for yourself. =)

**Comment by matthew-brett** [comment]
I stand corrected ...

**Pull request merged**